classes/classes/Scenes/Areas/GlacialRift/Amarok.as

```diff
@@ -1,57 +1,47 @@
-// By Foxwells
-// Amarok, a big ass wolf from Inuit mythology. Idek. Glacial Rift is depressingly empty
-// Since it's a big ass god wolf, it's pretty hard to beat
-// Herm, since there's no set gender for it in its mythology
-// Comes with a unique bad end if you lose to it too many times, kinda like the Phoenix
-
-package classes.Scenes.Areas.GlacialRift 
-{
+//By Foxwells
+//Amarok, a big ass wolf from Inuit mythology. Idek. Glacial Rift is depressingly empty
+//Since it's a big ass god wolf, it's pretty hard to beat
+//Herm, since there's no set gender for it in its mythology
+//Comes with an unique bad end if you lose to it too many times, kinda like the Phoenix
+package classes.Scenes.Areas.GlacialRift {
 	import classes.*;
 	import classes.BodyParts.*;
 	import classes.BodyParts.Hips;
 	import classes.internals.WeightedDrop;
-	
-	public class Amarok extends Monster
-	{
-		
+
+	public class Amarok extends Monster {
+
 		protected function amarokClaw():void {
-			//Blind
-			if(hasStatusEffect(StatusEffects.Blind)) {
+			if(hasStatusEffect(StatusEffects.Blind)) { //blind
 				outputText("The Amarok lunges for you, attempting to slash you with one of its paws, but misses completely due to its blindness.");
 				combatRoundOver();
 			}
-			//Dodge that shit yo
-			if (player.getEvasionRoll()) {
+			if (player.getEvasionRoll()) { //dodge that shit yo
 				outputText("The Amarok throws itself at you, attempting to slash you with its claws. Luckily, you manage to move out of the way.");
 				combatRoundOver();
 			}
 			else {
-				//Damage roll
 				outputText("The Amarok throws itself at you and rakes one of its hefty paws across you. Its claws slice you open and draw blood.");
 				var damage:int = ((str + 50) + rand(100));
 				damage = player.reduceDamage(damage);
-				player.takeDamage(damage, true);
+				player.takeDamage(damage, true); //damage roll
```

Comment:
This is one of the examples, where the change to the comment should be **reverted to its original state**.
The comment stands for the 'Damage roll 'block'. You've moved it 'after' the expression, where the player takes the already calced damage.